subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -36,6 +36,15 @@ config BT_LLL_VENDOR_NORDIC
 	select BT_CTLR_XTAL_ADVANCED_SUPPORT
 	select BT_CTLR_SCHED_ADVANCED_SUPPORT
 	select BT_CTLR_TIFS_HW_SUPPORT
+	# Direction Finding
+	select BT_CTLR_DF_CTE_TX_SUPPORT if SOC_NRF52833 && \
```

Comment:
Instead of ` if SOC_NRF52833`, create a new config similar to `HAS_HW_NRF_RADIO_BLE_2M` and `HAS_HW_NRF_RADIO_BLE_CODED`? That scales better to new HW platforms and simulated targets